boot/boot_test.go

```diff
@@ -764,7 +764,7 @@ func (s *bootenv20Suite) TestCoreParticipant20SetNextSameKernelSnap(c *C) {
 	c.Assert(bootKern.IsTrivial(), Equals, false)
 
 	// make the kernel used on next boot
-	rebootRequired, err := bootKern.SetNextBoot()
+	rebootRequired, err := bootKern.SetNextBoot(boot.NextBootContext{BootWithoutTry: false})
```

Comment:
(nitpick^2) fwiw, this could just be ` rebootRequired, err := ootKern.SetNextBoot(boot.NextBootContext{})` as go initializes the BootWithoutTry as false by default.